app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, vstr: dict) -> str:
+        if "vaccine" not in vstr.keys() or vstr["vaccine"] == None:
```

Comment:
1. None is the only instance of a NoneType object, so checking for None instead of == None is faster and more reliable.
2. It is much better to use the dictionary method "get" to avoid a KeyError if the key "vaccine" does not exist
This line can be completely replaced with if not vstr.get("vaccine"):